AK Zeus/app/src/main/java/ayds/zeus/songinfo/home/view/SongToReleaseDateStringMapper.kt

```diff
@@ -0,0 +1,59 @@
+package ayds.zeus.songinfo.home.view
+import ayds.zeus.songinfo.home.model.entities.Song
+
+
+interface SongToReleaseDateStringMapper {
+    fun map(song: Song): String
+}
+
+internal class SongToGenericReleaseDateStringMapper:SongToReleaseDateStringMapper{
+    override fun map(song:Song):String{
+        return song.releaseDate
+    }
+}
```

Comment:
Se puede eliminar el parámetro `song` de la función map, ya que cuando creamos el maper necesitamos ese mismo objeto

```suggestion
interface SongToReleaseDateStringMapper {
    fun map(): String
}

internal class SongToGenericReleaseDateStringMapper(private song: Song) :SongToReleaseDateStringMapper{
    override fun map():String{
        return song.releaseDate
    }
}
```